rest/config.go

```diff
@@ -1948,7 +1948,7 @@ func (sc *ServerContext) fetchConfigsSince(ctx context.Context, refreshInterval
 		sc.fetchConfigsLastUpdate = time.Now()
 	}
 
-	return sc.dbConfigs, nil
+	return sc._dbConfigs, nil
```

Comment:
This function is weird because it returns a value that would require you to hold a lock. This behavior should be noted in the docstring and considered.

I didn't look through the implications of this.